js/src/components/free-listings/configure-product-listings/checkErrors.test.js

```diff
@@ -73,15 +75,20 @@ describe( 'checkErrors', () => {
 			expect( errors.shipping_rate ).toMatchSnapshot();
 
 			// Invalid value
-			errors = checkErrors( { shipping_rate: 'automatic' }, [], [], [] );
```

Comment:
I checked the git log history and thought of asking you about it, but I figured you probably named it `'automatic'` because it is the opposite of the existing value `'manual'`. 😆